src/ledctl/Makefile.am

```diff
@@ -21,4 +21,4 @@ ledctl_SOURCES = ledctl.c
 
 ledctl_LDADD = ../lib/libledinternal.la ../common/libcommon.la
 ledctl_CFLAGS = -I$(top_srcdir)/src/lib/include -I$(top_srcdir)/src -I$(top_srcdir)/config \
-        -I$(top_srcdir)/src/lib $(AM_CFLAGS)
+        -I$(top_srcdir)/src/lib $(AM_CFLAGS) $(ENABLE_TEST)
```

Comment:
Please remove passing ENABLE_TEST flag to lib and remove checking this flag with ifdef from lib implementation.
It shall be considered always. Please keep checking this flag for ledctl.